languages/en.ini

```diff
@@ -1100,6 +1100,12 @@ Series = "Series"
 Set = "Set"
 shortlink_redirect = "You will be redirected in %%delay%% seconds..."
 show_filters_html = "Show filters (%%count%%)"
+show_less = "Show less"
```

Comment:
Are the show_less / show_more translations actually used anywhere? If not, can we remove them?